src/math/math.js

```diff
@@ -35,18 +37,18 @@ let getFib = (arg_n) => {
  * @param {Number} n n'th natural number
  * @returns return a `series` of n'th fibonacci numbers.
  */
-let printFib = (n) => {
+ let printFib = (n) => {
     let num1 = 1;
     let num2 = 0;
     let num3 = 0;
-    let st = "";
+    let st = [];
     for(let i=0;i<n;i++){
         num3 = num1 + num2;
-        st = st + num3 + ' ';
+        st.push(num3);
         num1 = num2;
         num2 = num3;
     }
-    return("Fibonacci Series : "+st);
+    return("Fibonacci Series : "+st.join(' '));
```

Comment:
I think there is no optimization here. and through the string.join() you have added another loop if you are not satisfied with my opinion please let me know.